components/developer/gcc-11/Makefile

```diff
@@ -56,16 +55,23 @@ CONFIGURE_LIBDIR.64 = $(CONFIGURE_PREFIX)/lib
 CONFIGURE_OPTIONS+= --with-diagnostics-urls=auto-if-env
 CONFIGURE_OPTIONS+= --enable-plugins
 CONFIGURE_OPTIONS+= --enable-objc-gc
-CONFIGURE_OPTIONS.i386+= --enable-initfini-array
+CONFIGURE_OPTIONS.sparc+= --with-mulhigh-size=2048
+CONFIGURE_OPTIONS.sparc+= --with-cpu=ultrasparc3
+CONFIGURE_OPTIONS.sparc+= --enable-float128
```

Comment:
Out of curiosity, is float128 enabled also on amd64?

Reply:
From the top of my head I think yes, per default.

Reply:
There is simply no support, __Float128 is defined nowhere.